index.bs

```diff
@@ -949,14 +950,20 @@ series of steps |steps|, run these steps:
 
 <div algorithm="termination-initiated-by-server">
 Whenever a [=WebTransport session=] which is associated with a {{WebTransport}} |transport| is
-[=session/terminated=], run these steps:
+[=session/terminated=] with optionally |code| and |reasonBytes|, run these steps:
 
 1. Let |cleanly| be a boolean representing whether the HTTP/3 stream associated with the CONNECT
    request that initiated |transport|'s [=[[Session]]=] is in the "Data Recvd" state. [[!QUIC]]
 1. [=Queue a network task=] with |transport| to run these steps:
   1. If |transport|'s [=[[State]]=] is `"closed"` or `"failed"`, abort these steps.
-  1. Let |reason| be TBD.
-  1. Let |error| be TBD.
+  1. Let |error| be the result of [=WebTransportError/creating=] a {{WebTransportError}} with
+     `"session"`.
+  1. Let |reason| be |error|.
+  1. If |cleanly| is true:
+     1. Set |reason| to a [=new=] {{WebTransportCloseInfo}}.
+     1. If |code| is given, set |reason|'s {{WebTransportCloseInfo/errorCode}} to |code|.
+     1. If |reasonBytes| is given, set |reason|'s {{WebTransportCloseInfo/reason}} to |reasonBytes|,
```

Comment:
Interesting question: what happens if reasonBytes can't be decoded as UTF-8? Is it rejected by the protocol layer?

Reply:
I expect so, but the discussion hasn't been settled. Please see https://github.com/ietf-wg-webtrans/draft-ietf-webtrans-http3/pull/60#discussion_r687405184. cc: @vasilvv